velox/functions/prestosql/window/tests/NthValueTest.cpp

```diff
@@ -333,6 +333,29 @@ TEST_F(NthValueTest, ignoreNulls) {
         createTable);
     createTable = false;
   }
+
+  input = makeRowVector({
```

Comment:
Nit : Maybe separate this into an individual test ?

Reply:
Hi @aditi-pandit, I've updated the unit test. Could you take another look again? Thanks!